tests/src/Functional/SettingsTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online\Functional;
+
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests the Collabora configuration.
+ */
+class SettingsTest extends BrowserTestBase {
```

Comment:
The test is mostly focused on the form, not on other functionality related to these settings.
So perhaps it should be called ConfigFormTest.

Reply:
Renamed